docs/5.1. Transport - MQTT.md

```diff
@@ -61,22 +85,51 @@ Example of IS-05 PATCH request on a receiver:
 
 ### 3.3 Disconnecting/Parking
 
-A disconnection IS-05 PATCH request should always trigger the receiver to unsubscribe from the associated `broker_topic`.
+A disconnection IS-05 PATCH request should always trigger the receiver to unsubscribe from the associated `broker_topic` and `connection_status_broker_topic`.
+If multiple receivers share a connection to the broker then it is up to the implementer to only unsubscribe from the `connection_status_broker_topic` when it is no longer needed by any managed receivers.
+
+### 3.4 Connection logic flow
+
+#### Step 1
+
+If the client does not currently have a connection to the MQTT broker indicated by `destination_host` and `destination_port` (for senders) or `source_host` and `source_port` (for receivers), it opens a connection.
+
+A sender client should specify a `connection_status` message with `"active"` set to false, as the Will Message in the CONNECT packet, using the `connection_status_broker_topic`.
```

Comment:
Is it worth quoting uses of 'Will Message' just to avoid confusion for anyone unfamiliar? Or link to the heading lower down.

Reply:
I have capitalised 'Will Message' , 'RETAIN flag' and 'CONNECT packet', etc. based on how they are written in the MQTT spec. We could turn all of these into links to the section of the MQTT spec or to a reference section in this doc, or just quote or italicise them?

Reply:
Italicising them may be enough. Having links for every occurrence may be overkill.